Caltech-IPAC/firefly
# DM-12555 query HiPS server to get available HiPS

The development includes,
- send query to HiPS server to get public HiPS
   per information from [http://aladin.u-strasbg.fr/hips/list](url) 
   the available HiPS images can be retrieved by sending the query like
  [http://alasky.unistra.fr/MocServer/query?hips_service_url=*&dataproduct_type=!catalog&dataproduct_type=!cube&get=record](url)
- show the list of the found public HiPS  in a table 

test:
 - go firefly/firefly-dev.html
 - select 'Test Searches', 
 - select 'HiPS', a table containing available HiPS is shown. 
 - highlight the HiPS to be searched, click 'search'
 - on the tri-view page, click 'Find HiPS', and get a popup with a table containing all available HiPS 
 - highlight the HiPS for another search.
